### DISCUSSION TITLE
Hide skins with controller

### DISCUSSION THREAD

**Discussion by newuser9290** [LIJI32/SameBoy#634] [Ideas]
I have purchased a backbone and controller and it works perfect with SameBoy. Can we have an option for buttons to disappear when controller is connected?

**Comment by newuser9290** [comment]
> I have purchased a backbone and controller and it works perfect with SameBoy. Can we have an option for buttons to disappear when controller is connected?

![image](https://github.com/LIJI32/SameBoy/assets/164794137/53b40329-3e3e-4d46-91f7-329f3995dac8)

**Comment by LIJI32** [comment]
Not yet, I'll look into it for the next version.
